src/lib/client.cc

```diff
@@ -128,16 +128,48 @@ Client::GetBucket(const QString& bucketName, const QString& prefix,
 void
 Client::CreateBucket(const QString& name)
 {
-	ds3_request* request = ds3_init_put_bucket(name.toUtf8().constData());
-	LOG_INFO("PUT          BUCKET    "+m_endpoint+"/"+name);
-	ds3_error* ds3Error = ds3_put_bucket(m_client, request);
-	ds3_free_request(request);
+    ds3_request* request = ds3_init_put_bucket(name.toUtf8().constData());
```

Comment:
Not intentionally, but it looks like the editor changed it to spaces. Messes up git diff, but aren't spaces preferred?